server-code/Changes.js

```diff
@@ -108,6 +108,17 @@ function getStatusUpdateContent(wbsNum) {
     return html;
 }
 
+/**
+* Gets the current status of the Work Package
+*
+* @return {Integer} - Status of the work package
+*/
+
+function getCurrentStatus(wbsNum) {
```

Comment:
Is this function currently being used? Or are you planning to do something with it in the future?

Reply:
Oop that function was just for testing and should be removed. I will correct this shortly thanks!